web/themes/custom/votegov/src/sass/components/quick-links.scss

```diff
@@ -0,0 +1,63 @@
+@use "uswds-core" as *;
+@use "variables" as *;
+
+.vote-quicklinks {
+  background-color: #11385B;
```

Comment:
Let me know if i need to add this to the variables file like the hero section... i found the uswds mixin for color but its a little bit different so wasnt sure if i need to do that or stick with the one listed in the design.

Reply:
Updated to $base-dark